astro-migration-provider/README.md

```diff
@@ -0,0 +1,60 @@
+# Astronomer Migration Provider
+
+Apache Airflow Provider containing Operators from Astronomer. The purpose of these operators is to better assist customers migrating to Astronomer hosted Airflow environments from MWAA, GCC, OSS
+
+## Installation
+Install and update using [pip](https://pip.pypa.io/en/stable/getting-started/):
+```text
+pip install https://astro-migration-provider.s3.us-west-2.amazonaws.com/astronomer-migration-provider-0.1.0.tar.gz
```

Comment:
🤞 pypi soon

Reply:
🤞yes! put this will allow us to test in the interim